BackfillPowerBI.yml

```diff
@@ -0,0 +1,58 @@
+parameters:
+- name: DeleteAllOrganizationData
+  displayName: Delete all data in Power BI Simeon Data source
+  default: false
+  type: boolean
+
+- name: BackfillHours
+  displayName: Number of hours to include in backfill
+  default: 72
+  type: number
+
+name: $(Date:yyyyMMdd)$(Rev:.r)
+trigger: none
+pool:
+  vmImage: windows-latest
+resources:
+  repositories:
+  - repository: Reporting
+    type: github
+    name: simeoncloud/reporting
+    ref: refs/heads/feature/uploadToPowerBI
+    endpoint: simeoncloud
+jobs:
+- job: Job
+  timeoutInMinutes: 60
```

Comment:
This will be enough for an org with 100 tenants?

Reply:
What do you suggest it be?

Reply:
I don't know how long is runs today....I defer to you to do the math on how long a backfill for 100 tenants would take...then multiply that by 10 I suppose

Reply:
Takes 2.5 minutes for 8 tenants to backfill 72 hours. Lets say 30 minutes for 100 tenants, maybe 3 hours?